src/agent/sqlite_manager/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.24)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_BUILD_TYPE RelWithDebInfo)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/../../vcpkg/scripts/buildsystems/vcpkg.cmake")
+set(VCPKG_MANIFEST_DIR ${CMAKE_SOURCE_DIR}/../../)
+
+project(SQLiteManager)
+
+find_package(SQLiteCpp REQUIRED)
+
+include_directories(include)
+
+add_library(SQLiteManager src/sqlite_manager.cpp)
+
+target_include_directories(SQLiteManager PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include PRIVATE ${SQLiteCpp_INCLUDE_DIRS})
+target_link_libraries(SQLiteManager PUBLIC SQLiteCpp)
+
+if(BUILD_TESTS)
+    # Enable testing
+    enable_testing()
+    add_subdirectory(tests)
+    #add_executable(SQLiteManager_test test/sqlite_manager_test.cpp src/sqlite_manager.cpp)
+    #target_include_directories(SQLiteManager_test PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${JWT_CPP_INCLUDE_DIRS})
+    #target_link_libraries(command_dispatcher_test PUBLIC ${Boost_LIBRARIES} OpenSSL::SSL OpenSSL::Crypto GTest::gtest GTest::gtest_main GTest::gmock GTest::gmock_main)
+endif()
```

Comment:
Add a new line at the end of the file. Validate this in all files.

Reply:
Done